lineage/data/figures/figure10.csv

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg width="100%" height="100%" viewBox="0 0 1280 800" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" xmlns:serif="http://www.serif.com/" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2;">
+    <g transform="matrix(1.81387,0,0,1.81387,629.322,532.495)">
+        <clipPath id="_clip1">
+            <rect x="-253.445" y="-268.719" width="501.496" height="424.315"/>
+        </clipPath>
+        <g clip-path="url(#_clip1)">
+            <g transform="matrix(1,0,0,1,-306,-396)">
```

Comment:
These files are very small... are you sure they have the cartoons?

Reply:
Yeah, I opened it on jupyter lab and they have the cartoons
<img width="1420" alt="Screen Shot 2020-05-28 at 12 42 06 PM" src="https://user-images.githubusercontent.com/54101546/83185971-a899ed80-a0e0-11ea-8c9b-d8ff01ad0b26.png">